ultralytics/nn/autobackend.py

```diff
@@ -193,7 +193,12 @@ def __init__(
             batch_dim = get_batch(ov_model)
             if batch_dim.is_static:
                 batch_size = batch_dim.get_length()
-            ov_compiled_model = core.compile_model(ov_model, device_name="AUTO")  # AUTO selects best available device
+            ov_compiled_model = core.compile_model(
+                ov_model,
+                device_name="AUTO",  # AUTO selects best available device
+                config={"PERFORMANCE_HINT": "THROUGHPUT"},
```

Comment:
Let's use CUMULATIVE_THROUGHPUT here.

And, of course, the config should be LATENCY for other cases, so an "if" is needed here.

Reply:
Yes, we need to default to LATENCY and then if-else to CUMULATIVE_THROUGHPUT

Reply:
Note to self modes:

- LATENCY
- THROUGHPUT (just 1 device, do not use)
- CUMULATIVE_THROUGHPUT (all devices)

Reply:
Ok I've unified the mode into a single variable, hardcoded to `LATENCY` currently but with plans to define this based on source dataset size.